Source/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -100,8 +100,11 @@ - (BOOL)presentExternalUserAgentRequest:(id<OIDExternalUserAgentRequest>)request
         if (callbackURL) {
           [strongSelf->_session resumeExternalUserAgentFlowWithURL:callbackURL];
         } else {
+          int errorCode;
+          NSString *descriptionOfTopView = [[[[[[UIApplication sharedApplication] keyWindow] subviews] lastObject] class] description];
```

Comment:
Unfortunately, this code is highly susceptible to failing in the future. It is walking a view hierarchy of code we don't own (Apple's implementation of `ASWebAuthenticationSession`).

I don't think we can use this approach.